src/Components/Aspire.AWS/Aspire.AWS.csproj

```diff
@@ -0,0 +1,24 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
The pattern we've been following is having a "component" library for each individual service. So I would imagine there would be one for S3, one for SQS, etc.

Reply:
Check out https://github.com/dotnet/aspire/pull/825 which is a good example of all the necessary steps to adding a full component.

Reply:
Thanks for the information @eerhardt, I'll fix it in the next iteration.